src/simulate.js

```diff
@@ -53,7 +52,37 @@ module.exports = function (opts, simHostOpts) {
     });
 };
 
-function parseStartPage(projectRoot) {
+var closeServer = function() {
+    return server.server && server.server.close();
+};
+
+var launchBrowser = function(target, url) {
+    return cordovaServe.launchBrowser({ target: target, url: url });
+};
+
+var simulate = function(opts) {
```

Comment:
I added a [new option, simhostui](https://github.com/Microsoft/cordova-simulate/pull/11/files/334e563898a5d13a539f575dcbebac6f4d6e711f#diff-6ddd72d4edcfc54d2078a3ddee13c0c7R19) - which currently supports an arbitrary UI directory, very similar to how the old `taco-simulate` and `taco-simulate-server` integration used to work. Please let me know if this answers your question. If we need to support more, or I missed something while porting over the changes, let's talk about it.